apps/browser-extension-wallet/src/providers/AnalyticsProvider/analyticsTracker/types.ts

```diff
@@ -132,13 +137,18 @@ export type PostHogActionsKeys =
   | 'DONE_GO_TO_WALLET';
 export type PostHogOnboardingActionsValueType = Partial<Record<PostHogActionsKeys, PostHogAction>>;
 export type PostHogOnboardingActionsType = Partial<Record<OnboardingFlows, PostHogOnboardingActionsValueType>>;
+export type PostHogSetMetadata = {
```

Comment:
I think `set` is how PostHog technically names it (pretty badly as it doesn't tell me much :) ) - shall we give it a bit more meaningful name indicating the real purpose? :) 
```suggestion
export type PostHogPersonProperties = {
```